packages/next-templates/src/app/productpagina-klachtenformulier/page.tsx

```diff
@@ -0,0 +1,164 @@
+'use client';
+
+import {
+  Article,
+  BreadcrumbNav,
+  BreadcrumbNavLink,
+  BreadcrumbNavSeparator,
+  ButtonLink,
+  ButtonGroup,
+  Heading1,
+  Heading2,
+  Link,
+  Paragraph,
+  Page,
+  PageContent,
+  SpotlightSection,
+  UnorderedList,
+  UnorderedListItem,
+} from '@utrecht/component-library-react';
+
+import { ExampleHeader } from '@/components/ExampleHeader/ExampleHeader';
+import { ExampleNavigation } from '@/components/ExampleNavigation/ExampleNavigation';
+import { ExampleFooter } from '@/components/ExampleFooter/ExampleFooter';
+import { PreFooterNav } from '@/components/PreFooterNav/PreFooterNav';
+import { IconChevronRight } from '@tabler/icons-react';
+
+import '@utrecht/design-tokens/dist/index.css';
+import '@/app/styling/css/productpagina-klachtenformulier.css';
+import '@/app/styling/css/detail-page.css';
+
+import { useEffect } from 'react';
```

Comment:
import zonder "side effects" moeten vóór de imports met side effects staan. De `.css` imports voeren na compilatie code uit om CSS te injecteren, en die hebben een side effect. Die moeten daarom altijd als laatste.

Meestal waarschuwt ESLint daarvoor ook, maar misschien ontbreekt in deze repo de `import` plugin voor ESLint.

Reply:
Good to know, dat wist ik niet! 

Ik heb hier binnen GV inderdaad geen warning over gekregen en heb deze volgorde overgenomen van een pagina in de wmebv map. Op verschillende pagina's is dit dan dus verkeerd toegepast. Ik zal hier een issue over aanmaken en heb het op deze pagina aangepast.